src/features/components/organisms/markets/greyed-container/trade-benefits.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { item_container } from './styles.module.scss'
+import { TradeBenefitType } from './types'
+import Flex from 'features/components/atoms/flex-box'
+import { Localize } from 'components/localization'
+import Typography from 'features/components/atoms/typography'
+import Image from 'features/components/atoms/image'
+
+interface TradeBenefitProps {
+    item: TradeBenefitType
+}
+
+const TradeBenefit = ({ item }: TradeBenefitProps) => {
+    return (
+        <Flex.Box className={item_container} direction="col" align="center" gap="5x">
+            <Image src={item.icon_src} width={48} height={48} />
+
+            <Flex.Item>
+                <Typography.Paragraph size="medium" textcolor="inverted" align="center">
```

Comment:
size='mediu
<img width="1063" alt="Screenshot 2023-05-31 at 17 40 31" src="https://github.com/binary-com/deriv-com/assets/103182473/c33d957f-7c6c-4dc0-b7f3-40e972fdb41a">
m' here by default


```suggestion
                <Typography.Paragraph textcolor="inverted" align="center">
```